CribblyBackend/Controllers/PlayerController.cs

```diff
@@ -18,6 +19,41 @@ public PlayerController(IPlayerService playerService)
             this.playerService = playerService;
         }
 
+        /// <summary>
+        /// Handles the login. Creates or gets the player specified in the request.
+        /// </summary>
+        /// <param name="request">The request</param>
+        /// <returns></returns>
+        [HttpPost("login")]
+        public async Task<IActionResult> Login([FromBody] LoginRequest request)
```

Comment:
Do you think we would need the Create endpoint anymore if a user is automatically created/registered when logging in?

Reply:
Nah, I'll get rid of it